CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Versioned according to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+Added:
+
+  * many utility methods for image manipulation and coordinate handling, #268
```

Comment:
Could we advertise this some more? After all, this is supposed to become the _recommended_ or even _required_ way of processing images on a workspace (when working with core). This should at least mention that it offers spec-conforming `AlternativeImage` processing. Also, we should refer to OCR-D/spec#116 (regarding the details that are still missing in the spec but already assumed here).